src/primitives/createLazyLoadQuery.ts

```diff
@@ -53,6 +53,7 @@ export function createLazyLoadQuery<TQuery extends OperationType>(
 	options?: {
 		fetchPolicy?: MaybeAccessor<FetchPolicy | undefined>;
 		networkCacheConfig?: MaybeAccessor<CacheConfig | undefined>;
+		deferStream?: MaybeAccessor<boolean | undefined>;
```

Comment:
`deferStream` is a SSR-only option so making it reactive has no effect. We should explicitly show that this option is not reactive through the type def therefore it should be just `boolean`